functions/ts/src/helm_inflator.ts

```diff
@@ -228,8 +248,7 @@ metadata:
     config.kubernetes.io/function: |
       container:
         image: gcr.io/kpt-functions/helm-inflator
-        network:
-          required: true
+        network: true
```

Comment:
Why format change?

Reply:
The `network` field in function config has been updated months ago but this example hasn't updated.